### ISSUE TITLE
utils.plot_imgs returns an empty table

### ISSUE THREAD

**Issue by EtagiBI** [karolzak/keras-unet#28]
Hello,

I'm trying to plot templates and masks using `utils.plot_imgs` function. Here's my code:
```
template = cv2.imread(r"E:\Templates\741499_1.jpg")
true_mask = cv2.imread(r"E:\Masks\741499_1.jpg")
pred_mask = cv2.imread(r"E:\Predictions\741499_1.jpg")
assert isinstance(template, (np.ndarray)) == isinstance(true_mask, (np.ndarray)) == isinstance(pred_mask, (np.ndarray))
plot_imgs(org_imgs=template, mask_imgs=true_mask, pred_imgs=pred_mask, nm_img_to_plot=1)
```

Unfortunately, I get an empty table as a result:
![изображение](https://user-images.githubusercontent.com/20855725/87428010-1fd90180-c5fb-11ea-96d9-cd366c343e55.png)

Am I missing something? My images are 512x512 RGB.

**Comment by karolzak** [comment]
Hi @EtagiBI 

You need to pass in these images into `plot_imgs` as arrays of arrays.
Also you might need to convert all the mask images into grayscale (single channel) images.
Try this:
```python
template = np.expand_dims(cv2.imread(r"E:\Templates\741499_1.jpg"), axis=0)
true_mask = np.expand_dims(cv2.imread(r"E:\Masks\741499_1.jpg", cv2.IMREAD_GRAYSCALE), axis=0)
pred_mask = np.expand_dims(cv2.imread(r"E:\Predictions\741499_1.jpg", cv2.IMREAD_GRAYSCALE), axis=0)
assert isinstance(template, (np.ndarray)) == isinstance(true_mask, (np.ndarray)) == isinstance(pred_mask, (np.ndarray))
plot_imgs(org_imgs=template, mask_imgs=true_mask, pred_imgs=pred_mask, nm_img_to_plot=1)
```

**Comment by EtagiBI** [comment]
It works indeed, thank you! 
In the documentation for `plot_imgs` function is says
```
 Args:
        org_imgs (numpy.ndarray): [description]
        mask_imgs (numpy.ndarray): [description]
        pred_imgs (numpy.ndarray, optional): [description]. Defaults to None.:
```
Could you add information about "array of arrays" requirement, please?

**Comment by karolzak** [comment]
@EtagiBI docstrings for `plot_imgs` are now fixed and more informative in version 0.1.2+
Thanks again for pointing that out
